1906-11-26.xml

```diff
@@ -0,0 +1,1934 @@
+                                    Marseilles</cell>
+                                <cell>£12.9.8</cell>
+                                <cell>£9.10.3</cell>
+                            </row>
+                            <row>
+                                <cell>From Alexandria To Port Said</cell>
+                                <cell>£1.15.10</cell>
+                                <cell>£1.7.10</cell>
+                            </row>
+                            <row>
+                                <cell>From Alexandria to Jaffa</cell>
+                                <cell>£3.3.5</cell>
+                                <cell>£2.2.5</cell>
+                            </row>
+                            <row>
+                                <cell>From Alexandria to Beyrouth</cell>
+                                <cell>£4.7.2</cell>
+                                <cell>£3.3.2.</cell>
+                            </row>
+                            <row>
+                                <cell>Through tickets for Paris (via Marseilles from
+                                    Alexandria)</cell>
+                                <cell>£15.12.1</cell>
+                                <cell>£10.12.5</cell>
+                            </row>
+                            <row>
+                                <cell>Through tickets for Paris (via Marseilles) from Port Said
+                                    (directly or via Alexandria)</cell>
+                                <cell>£16.5.11</cell>
+                                <cell>£12.1.5</cell>
+                            </row>
+                            <row>
+                                <cell>Through tickets for London (via Marseilles) (Calais-Douvree)
+                                    from Alexandria or Port Said (directly or via Alexandria)</cell>
+                                <cell>£16.12.10</cell>
+                                <cell>£12.9.8</cell>
+                            </row>
+                            <row>
+                                <cell>Interchangeable return tickets with the Austrian Lloyd Cy.
+                                    (available one way by Messageries</cell>
+                                <cell>£21.11.10</cell>
+                                <cell>£15.11.2</cell>
+                            </row>
+                        </table>
+                        <table rend="frame" xml:id="SailingfromPortSaid">
+                            <head><hi rend="bold">Sailing from Port Said in November,
+                                1906</hi></head>
+                            <row>
+                                <cell rows="5">For Marseilles Direct</cell>
+                                <cell>Probably on</cell>
+                                <cell>Thursday</cell>
+                                <cell><date when="1905-07-06">8 Nov</date></cell>
```

Comment:
this date and a bunch that follow in this ad need to be updated--in this case to when="1906-11-08"